File: rust/src/models/never_reset_period.rs
// this file is @generated
use serde::{Deserialize, Serialize};

/// Never resets — counts all usage since the subscription was activated.
#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct NeverResetPeriod {}

impl NeverResetPeriod {
    pub fn new() -> Self {
        Self {}
    }
}

File: browser/src/models/neverResetPeriod.ts
/** Never resets — counts all usage since the subscription was activated. */
export interface NeverResetPeriod {}

export const NeverResetPeriodSerializer = {
  _fromJsonObject(object: any): NeverResetPeriod {
    return {};
  },

  _toJsonObject(self: NeverResetPeriod): any {
    return {};
  },
};
